docs/example/general.rst

```diff
@@ -169,7 +169,7 @@ specific test environment section, for example:
 .. code-block:: ini
 
     [testenv:docs]
-    basepython = python2.7
+    basepython = python2.7  # or C:\mypath\pythons\3.9\python.exe
```

Comment:
Hard coding the path is never a good idea, and with newer python installations registering themselves into the registry we can easily discover them. I'd not document behaviour that is not considered best practice.

Reply:
I agree. The thought was to demonstrate that you can point to the actual interpreter for specific environments;